packages/core/tests/oob.test.ts

```diff
@@ -464,34 +470,42 @@ describe('out of band', () => {
       // Create first connection
       const outOfBandRecord = await faberAgent.oob.createInvitation(makeConnectionConfig)
       let { connectionRecord: firstAliceFaberConnection } = await aliceAgent.oob.receiveInvitation(
-        outOfBandRecord.outOfBandInvitation
+        outOfBandRecord.outOfBandInvitation!
       )
       firstAliceFaberConnection = await aliceAgent.connections.returnWhenIsConnected(firstAliceFaberConnection!.id)
 
-      const [firstFaberAliceConnection] = await faberAgent.connections.findAllByOutOfBandId(outOfBandRecord.id)
+      const [firstFaberAliceConnection] = await faberAgent.connections.findAllByOutOfBandId(outOfBandRecord!.id)
 
       // Create second connection
       const outOfBandRecord2 = await faberAgent.oob.createInvitation(makeConnectionConfig)
 
       // Take over the recipientKeys from the first invitation so they match when encoded
-      const [firstInvitationService] = outOfBandRecord.outOfBandInvitation.getInlineServices()
-      const [secondInvitationService] = outOfBandRecord2.outOfBandInvitation.getInlineServices()
+
+      expect(outOfBandRecord!.outOfBandInvitation).toBeInstanceOf(OutOfBandInvitation)
+      expect(outOfBandRecord2!.outOfBandInvitation).toBeInstanceOf(OutOfBandInvitation)
+
+      const outOfBandInvitation = outOfBandRecord!.outOfBandInvitation as OutOfBandInvitation
+      const outOfBandInvitation2 = outOfBandRecord2!.outOfBandInvitation as OutOfBandInvitation
+
+      const [firstInvitationService] = outOfBandInvitation.getInlineServices()
+      const [secondInvitationService] = outOfBandInvitation2.getInlineServices()
       secondInvitationService.recipientKeys = firstInvitationService.recipientKeys
 
       aliceAgent.events.on(OutOfBandEventTypes.HandshakeReused, aliceReuseListener)
       faberAgent.events.on(OutOfBandEventTypes.HandshakeReused, faberReuseListener)
 
-      const {
-        connectionRecord: secondAliceFaberConnection,
-        outOfBandRecord: { id: secondOobRecordId },
-      } = await aliceAgent.oob.receiveInvitation(outOfBandRecord2.outOfBandInvitation, { reuseConnection: true })
+      const { connectionRecord: secondAliceFaberConnection, outOfBandRecord: secondRecord } =
+        await aliceAgent.oob.receiveInvitation(outOfBandRecord2!.outOfBandInvitation!, { reuseConnection: true })
```

Comment:
Seems more exclamation marks here are not needed

Reply:
Fixed this file